tests/test.c

```diff
@@ -1,6 +1,6 @@
 #include "test.h"
 
-#define TEMP_FILE "temp.txt"
+#define TEMP_FILE "/tmp/temp_logs.txt"
```

Comment:
Yes, good idea--if we do this though, we'll also need to change the check in the SIGINT handler in `test.sh` to reflect this!